src/components/GeneralInfo/BootcImageCard/BootcImageCard.js

```diff
@@ -22,6 +22,13 @@ export const BootcImageCard = ({ handleClick, extra }) => {
         ...[{ title: 'Booted Image Digest', value: bootc.bootedImageDigest }],
         ...[{ title: 'Staged Image', value: bootc.stagedImage }],
         ...[{ title: 'Staged Image Digest', value: bootc.stagedImageDigest }],
+        ...[{ title: 'Available Image', value: bootc.availableImage }],
+        ...[
```

Comment:
It is not necessary to wrap each item in an array and spread it into the array, with `...[ ]`. Only the result of `extra.map()` needs to be spread, since map returns an array.

Reply:
Agreed, just following the conventions and style of the other code in the repo.

Reply:
@dkuc Though, never hesitate to break conventions if they aren't right. :)